tests/LocationControllerTest.cs

```diff
@@ -14,12 +14,19 @@ namespace BDMS;
 [TestClass]
 public class LocationControllerTest
 {
+    private const int BoreholeId1 = 2_000_000;
+    private const int BoreholeId2 = 2_000_001;
+    private const int BoreholeId3 = 2_000_002;
+
     private BdmsContext context;
     private LocationController controller;
     private Mock<IHttpClientFactory> httpClientFactoryMock;
     private Mock<ILogger<LocationController>> loggerMock;
     private Mock<ILogger<LocationService>> loggerLocationServiceMock;
 
+    private int boreholeCount;
+    private List<int> boreholeIds = new List<int>() { BoreholeId1, BoreholeId2, BoreholeId3 };
```

Comment:
PP: Hier können die Klammern auch weggelassen werden:
```suggestion
    private List<int> boreholeIds = new List<int> { BoreholeId1, BoreholeId2, BoreholeId3 };
```